.github/CODE_OF_CONDUCT.md

```diff
@@ -1,3 +1,9 @@
 # Code of Conduct
 
+If you use an LLM (Large Language Model, like ChatGPT, Claude, GitHub Copilot,
+or Llama) to make a contribution then you must say so in your contribution and
+you must carefully review your contribution for correctness before sharing it.
+If you share un-reviewed LLM-generated content then you will be immediately
+banned.
```

Comment:
This may be worth expanding on a bit more. Aside from the correctness issue, there is also the appropriate attribution and possible copyright washing involved with LLM-generated content.

This text is also *slightly* different than the proposed policy in the discussion, and I think that the discussion policy is more reasonable.

Reply:
Thanks, done.